localstack/utils/common.py

```diff
@@ -860,20 +863,33 @@ def rm_rf(path):
         shutil.rmtree(path)
 
 
-def cp_r(src, dst, rm_dest_on_conflict=False):
+def cp_r(src, dst, rm_dest_on_conflict=False, ignore_copystat_errors=False, **kwargs):
     """Recursively copies file/directory"""
-    if os.path.isfile(src):
-        return shutil.copy(src, dst)
-    kwargs = {}
-    if "dirs_exist_ok" in inspect.getfullargspec(shutil.copytree).args:
-        kwargs["dirs_exist_ok"] = True
+    # attention: this patch is not threadsafe
+    copystat_orig = shutil.copystat
+    if ignore_copystat_errors:
+
+        def _copystat(*args, **kwargs):
+            try:
+                return copystat_orig(*args, **kwargs)
+            except Exception as e:
+                print("ERROR (ignored): %s" % e)
```

Comment:
Let's better put a `pass` here instead of `print()` (frankly, I think even a `LOG.debug` would be too verbose, as this function is iterating through a potentially large directory tree).